ant/.ant.properties

```diff
@@ -14,7 +14,7 @@ jcommon.jar=${itext.lib}/jcommon.jar
 servlet.jar=${itext.lib}/servlet.jar
 
 bc.jdk=jdk15on
-bc.version=1.52
+bc.version=1.62
```

Comment:
the problematic version is 1.68, isnt it?

Reply:
The problematic version is indeed 1.68. However we do not use the ANT build. 
The change 1.52 -> 1.62 occurred between JS6 -> LS7 so I merged it.